bin/compile

```diff
@@ -86,23 +86,17 @@ fi
 
 # Install GPG key
 topic "Install gpg key for Datadog APT Repository"
-APT_KEYRING="$CACHE_DIR/apt/trusted.gpg"
-APT_KEYBOX="$CACHE_DIR/apt/temp.gpg"
-GPG_KEY_FILE="$BUILDPACK_DIR/etc/datadog.gpg"
+DATADOG_APT_KEYRING="$BUILDPACK_DIR/datadog-archive-keyring.gpg"
+DATADOG_APT_KEYS=("DATADOG_APT_KEY_CURRENT.public" "DATADOG_APT_KEY_F14F620E.public" "DATADOG_APT_KEY_382E94DE.public")
+DATADOG_APT_KEYS_URL="https://keys.datadoghq.com"
 GPG_HOME_DIR="$BUILD_DIR/.gnupg"
 mkdir -p "$GPG_HOME_DIR"
+touch $DATADOG_APT_KEYRING
 
-# Remove old APT keyrings to avoid type conflicts
-if [ -f $APT_KEYRING ]; then
-  rm $APT_KEYRING
-fi
-
-if [[ $STACK == "heroku-16" ]]; then
-  gpg --ignore-time-conflict --no-options --no-default-keyring --homedir "$GPG_HOME_DIR" --keyring "$APT_KEYRING" --import "$GPG_KEY_FILE" | indent
-else
-  gpg --ignore-time-conflict --no-options --no-default-keyring --homedir "$GPG_HOME_DIR" --keyring "$APT_KEYBOX" --trustdb "$APT_KEYRING" --import "$GPG_KEY_FILE" | indent
-  gpg --no-default-keyring --keyring "$APT_KEYBOX" --export > $APT_KEYRING
-fi
+for key in "${DATADOG_APT_KEYS[@]}"; do
+  curl -o "/tmp/${key}" "${DATADOG_APT_KEYS_URL}/${key}" | indent
+  gpg --ignore-time-conflict --no-options --no-default-keyring --homedir "$GPG_HOME_DIR" --keyring "$DATADOG_APT_KEYRING" --import "/tmp/${key}" | indent
```

Comment:
What are those `| indent` doing everywhere? 😂

Reply:
These are used throughout the script to indent all commands that have some output to make it look prettier + consistent when running the buildpack.

Reply:
Didn't know the command 👌

Reply:
FWIW it's not a command, it's a function implemented in `bin/common.sh`